### PR TITLE
Adds trialing label and sync infos with Stripe

### PR DESCRIPTION
**Pull request by YoruNoHikage**
# Plan
## No plan, no credits
plan-no-credits
## No plan, credits
plan-credits
## Trialing
plan-trialing
## Plan canceled
plan-canceled

# Cards (below plan/credits)
## Empty
cards-empty
## One card
cards

This component could be interesting as a story, we should consider it.

### CODE REVIEW

**Comment by yannickmathey** [comment]
We should change the color of trial label from grey to black or orange (contrats issue)

**Comment by yannickmathey** [comment]
When a user is under trial but has not credit card registered, the message should be: "Your plan will be ended at XX/XX/XXXX because you don't have a credit card registered. Would you want to add one?" – or something like that

**Comment by yannickmathey** [comment]
Plan canceled: It should display (in addition with the message under the input) a label in the same style of the trial one

File: app/styles/components/account/account-subscription.scss [hunk 1]
```diff
@@ -1,3 +1,20 @@
+@import '../../variables.scss';
+
+.account-subscription {
+	&-plan {
+		& .badge {
+			border: 2px solid $light-grey;
+			border-radius: 5px;
+			color: $light-grey;
```
Comment (FranzPoize) [inline app/styles/components/account/account-subscription.scss]: change in color as per yannick's recommandation

**Pull request merged**